pkgs/by-name/xo/xonsh/xontribs/xontrib-gitinfo/default.nix

```diff
@@ -0,0 +1,47 @@
+{
+  pkgs,
+  python3,
+}: let
+  pname = "xontrib-gitinfo";
+  version = "0.1.0";
+in
+  python3.pkgs.buildPythonPackage {
+    inherit pname version;
+
+    src = pkgs.fetchFromGitHub {
+      owner = "dyuri";
+      repo = "xontrib-gitinfo";
+      rev = "b1ba458d85a6684088807d962b39980144685630";
+      sha256 = "sha256-e5lgfcrG8p/3YgYNlnkfZIYj3VEjuNTRoseAl+Uyfd8=";
+    };
+
+    doCheck = false;
+
+    nativeBuildInputs = with pkgs; [
+      python3Packages.setuptools
+      python3Packages.wheel
+    ];
+
+    patchPhase = ''
```

Comment:
Is echoing out a setup.py better than a pyproject.toml?

Reply:
Removed from my branch